app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/check_all_answers.scala.html

```diff
@@ -182,6 +183,32 @@ <h2 class="heading-medium">@messages("task-list.upload-supporting-docs.link")</h
 
     }
 
+    case multipleIndirect: CompleteMultipleIndirectDisposalReturn => {
+        @multipleIndirect.representeeAnswers.map { representeeAnswers =>
+            <h2 class="heading-medium">@messages("task-list.representee.link")</h2>
+        @representeeSummary(
+            representeeAnswers,
+            Either.fromOption(representeeAnswers.dateOfDeath.map(_ => PersonalRepresentative), Capacitor).swap
+        )
+        }
+
+        <h2 class="heading-medium" >@messages("task-list.triage.link")</h2>
+        @multipleDisposalsTriageSummary(multipleIndirect.triageAnswers, isATrust)
+
+        <h2 class="heading-medium">Example company details</h2>
+        <h1>TO DO</h1>
```

Comment:
this is, unsurprisingly, to do - it can only be done once the summary template is defined for this section which is currently in progress